easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-4.4.5-intel-2015b-ParMETIS-4.0.3.eb

```diff
@@ -0,0 +1,25 @@
+name = 'SuiteSparse'
+version = '4.4.5'
+
+homepage = 'http://www.cise.ufl.edu/research/sparse/SuiteSparse/'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2015b'}
+toolchainopts = {'opt': True, 'unroll': True, 'pic': True}
+
+source_urls = ['http://faculty.cse.tamu.edu/davis/SuiteSparse/']
+sources = [SOURCE_TAR_GZ]
+
+patches = [
+           'AMD_GNUMakefile-4.4.5.patch', 
+           'UMFPACK_GNUMakefile-4.4.5.patch',
+          ]
```

Comment:
please fix indentation, use `%(version)s` and rename the patch files to start with `SuiteSparse-%(version)s`, i.e.:

```python
patches = [
    'SuiteSparse-%(version)s_AMD_GNUMakefile.patch', 
    'SuiteSparse-%(version)s_UMFPACK_GNUMakefile.patch',
]
```